lib/components/molecules/Table/types/index.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import type { Node } from 'react';
+
+type Content =
```

Comment:
Will your table always accept these props only? Its content you can't define that with component since use-cases can be limitless